src/field_theory/is_alg_closed/classification.lean

```diff
@@ -177,7 +177,7 @@ begin
 end
 
 private lemma ring_equiv_of_cardinal_eq_of_char_p (p : ℕ) [fact p.prime]
-  [char_p K p] [char_p L p] (hK : ℵ₀ < #K) (hKL : #K = #L) : K ≃+* L :=
+  [algebra (zmod p) K] [algebra (zmod p) L] (hK : ℵ₀ < #K) (hKL : #K = #L) : K ≃+* L :=
```

Comment:
I don't think we should change this one, since there's no definitional equality to care about

Reply:
After some thoughts I agree `char_p` is better here.